sql/flow_config.go

```diff
@@ -0,0 +1,77 @@
+package sql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type pypiVersionResponse struct {
+	Info struct {
+		Version string `json:"version"`
+	} `json:"info"`
+}
+
+type configResponse struct {
+	BaseDockerImage string `json:"baseDockerImage"`
+}
+
+const (
+	astroSQLCliProjectURL = "https://pypi.org/pypi/astro-sql-cli/json"
+	astroSQLCliConfigURL  = "https://raw.githubusercontent.com/astronomer/astro-sdk/astro-cli/sql-cli/config/astro-cli.json"
+	defaultDockerImageURI = "quay.io/astronomer/astro-runtime:6.0.4-base"
+)
+
+var (
+	getPypiVersion        = GetPypiVersion
+	getBaseDockerImageURI = GetBaseDockerImageURI
+)
+
+func GetPypiVersion(projectURL string) (string, error) {
+	httpClient := &http.Client{}
+	req, err := http.NewRequestWithContext(context.TODO(), http.MethodGet, projectURL, http.NoBody)
+	if err != nil {
+		err = fmt.Errorf("error creating HTTP request %w", err)
+		return "", err
```

Comment:
```suggestion
		return "", fmt.Errorf("error creating HTTP request %w", err)
```
nit: slightly simpler way to do the same thing, will leave it up to you on whichever way you want to go with it